packages/agents/chain-abstraction/src/helpers/swapdata.ts

```diff
@@ -38,14 +47,33 @@ export const getOriginSwapDataForUniV3 = async (_args: OriginSwapDataCallbackArg
  * including a function signature for the 1inch aggregator.
  */
 export const getOriginSwapDataForOneInch = async (args: OriginSwapDataCallbackArgs): Promise<string> => {
+  if (!args.config) throw new Error("No Authorizartion config provided for One Inch.");
```

Comment:
The following lines says both params: `customURL` and `apiKey` can be optional. then we can update it like this
```
  config?: {
    customURL?: string;
    apiKey?: string;
  }; 
```
If both are not provided here, throw like above: `if (!args.config) throw new ...`
```
if (!args.config.customURL && !args.config.apiKey) throw new Error("...")
```